validphys2/src/validphys/inconsistent_ct.py

```diff
@@ -0,0 +1,162 @@
+import dataclasses
+from validphys.coredata import CommonData
+import pandas as pd
+
+
+
+@dataclasses.dataclass(eq = False)
+class InconsistentCommonData(CommonData):
+    """
```

Comment:
Since you introduced this class it should be possible to remove the same new methods (with_MULT_sys etc.) from the CommonData class defined in coredata.py right?

Reply:
Yes that is the idea of this modifications. However, right now, the CommonData 'inconsistent' methods are still needed (see filters.py)

Reply:
Yes I know, it was just to know if this was the idea of the modifications, thanks.

Reply:
> Since you introduced this class it should be possible to remove the same new methods (with_MULT_sys etc.) from the CommonData class defined in coredata.py right?

They have been removed now :)